test/Microsoft.AspNetCore.Razor.Runtime.Test/Microsoft.AspNetCore.Razor.Runtime.Test.csproj

```diff
@@ -7,10 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="**\*.cs" />
+    <Compile Include="**\*.cs" Exclude="TestFiles\**\*" />
     <Compile Include="..\..\shared\Microsoft.AspNetCore.Razor.TagHelpers.Testing.Sources\**\*.cs" />
-    <Content Include="TestFiles\**\*" CopyToOutputDirectory="Always" />
-    <EmbeddedResource Include="TestFiles\**\*" />
```

Comment:
It isn't in the PJ version: https://github.com/aspnet/Razor/blob/e2dd09c918cfc9e77ae8f31042b1c65f94935e5a/test/Microsoft.AspNetCore.Razor.Runtime.Test/project.json

Also, this is actually part of what was causing https://github.com/dotnet/sdk/issues/518